### ISSUE TITLE
Unable to render a SCSS file located in `files` directory tree

### ISSUE THREAD

**Issue by cedric-anne** [glpi-project/glpi#15398]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

10.0.9 / main

### Bug description

`Html::compileScss()` method does not allow rendering SCSS files located outside the `GLPI_ROOT` directory structure, see
https://github.com/glpi-project/glpi/blob/0b9c9df222c06510a8ffcdb2884ffa42243751b0/src/Html.php#L6767-L6772

On GLPI main branch, it breaks the custom theme support (#11669) and on GLPI 10.0, it does not allow any plugin to render a scss file created under the `GLPI_PLUGIN_DOC_DIR`, as long as the `GLPI_VAR_DIR` is moved outside the GLPI root directory structure (as recommended in official documentation https://glpi-install.readthedocs.io/en/latest/install/index.html#files-and-directories-locations).

### Relevant log output

```shell
glpiphplog.WARNING:   *** PHP User Warning (512): Requested file /var/lib/glpi/custom_palette.scss is outside GLPI file tree. in /var/www/glpi/src/Html.php at line 6768
  Backtrace :
  src/Html.php:6768                                  trigger_error()
  front/css.php:60                                   Html::compileScss()
  public/index.php:82                                require()
```

### Steps To reproduce

1. Configure `GLPI_THEMES_DIR` to be located ouside the GLPI rott directory structure.
2. Create a `custom_palette.scss` file in it (content can be a copy of an existing theme from `css/palettes`).
3. Go to your preferences, select this custom theme, and save.
4. See that the theme is not rendered.

### Your GLPI setup information

GLPI 10.1.0-dev-git-feature/consumables-search-UX-d54101ed3f ( => /var/www/glpi)
Installation mode: GIT
Current language:en_GB

**Comment by AdrienClairembault** [comment]
Proposed solutions:
* #15402
* #15403

**Comment by cedric-anne** [comment]
Fixed by #15402.
